field/composite.go

```diff
@@ -295,6 +307,12 @@ func (f *Composite) Bytes() ([]byte, error) {
 	return f.pack()
 }
 
+// Bitmap returns the parsed bitmap instantiated on the key "0" of the spec.
+// In case the bitmap is not instantiated on the spec, returns nil.
+func (f *Composite) Bitmap() *Bitmap {
+	return f.spec.Bitmap
```

Comment:
should it be `f.Bitmap`?

Reply:
Done, I made a deep copy of spec.Bitmap and saved it to f.Bitmap in case f.Bitmap is nil, is this ok?